crypto/asn1/standard_methods.h

```diff
@@ -59,3 +59,61 @@ static const EVP_PKEY_ASN1_METHOD *standard_methods[] = {
 #endif
 };
 
+static const EVP_PKEY_ASN1_METHOD *standard_methods_fips[] = {
```

Comment:
These are all the supported FIPS-140-3 methods or just the ones we're actually going to submit?

Reply:
These are the ones we support - matches our published security policy.